cypress/integration/pane_spec.js

```diff
@@ -233,12 +233,12 @@ describe('Pane functionality', function() {
     cy.contains('View Account').click();
     cy.contains('Modify Account').click();
 
-    cy.get('#tag').then(($tag) => {
+    cy.get(`${qsPaneContent} .tag`).then(($tag) => {
       cy.get('input[type=text]').clear().type('Shane');
       cy.get('form').submit();
       cy.get(qsPaneCloseBtn).click();
 
-      cy.get('#tag').invoke('text').should('not.equal', $tag.text());
+      cy.get(`${qsPaneContent} .tag`).invoke('text').should('not.equal', $tag.text());
```

Comment:
There's a ton of repetition with `${qsPaneContent} .tag`. Thoughts on making that a variable?

Reply:
sure